composer.json

```diff
@@ -11,10 +11,14 @@
         {
             "type": "git",
             "url": "https://github.com/jszanto/gewisweb-test-data"
+        },
+        {
+            "type": "git",
+            "url": "https://github.com/laminas/laminas-code"
         }
```

Comment:
Would it not have been easier to just specify that we require `4.5.x-dev`, instead of explicitly cloning the repository and hoping that the `4.5.x` branch is the current main branch?

Reply:
If there's no release for 4.5.x-dev, will composer even be able to find the repository? Either way, I intent to remove this before merging.

Reply:
Yes, because Packagist knows about the `v4.5.x-dev` "release", which is essentially a placeholder for "use the branch v4.5.x".

```sh
tom@vm$ php composer.phar require laminas/laminas-code:v4.5.x-dev
./composer.json has been created
Running composer update laminas/laminas-code
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking laminas/laminas-code (4.5.x-dev 7fcc8ea)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Downloading laminas/laminas-code (4.5.x-dev 7fcc8ea)
  - Installing laminas/laminas-code (4.5.x-dev 7fcc8ea): Extracting archive
```

As long as it is removed before actually upgrading this is not an issue.